lib/cheetah.rb

```diff
@@ -455,54 +467,70 @@ def build_recorder(options)
       end
     end
 
-    def fork_commands_recursive(commands, pipes)
+    # Reopen *stream* to write **into** the writing half of *pipe*
+    # and close the reading half of *pipe*.
+    # @param pipe [Array<IO>] a pair of IOs as returned from IO.pipe
+    # @param stream [IO]
+    def into_pipe(stream, pipe)
```

Comment:
This is probably nitpicking, but for a method called `into_pipe`  I expect `pipe` to be the first argument (and same with `from_pipe`). As said, just NP, but it indeed puzzled me for some seconds.